src/lib/Backpack/devBackpack.cpp

```diff
@@ -59,7 +59,6 @@ void startPassthrough()
     delay(100);
     digitalWrite(GPIO_PIN_BACKPACK_EN, HIGH);
     delay(50);
-    digitalWrite(GPIO_PIN_BACKPACK_BOOT, LOW);
```

Comment:
The PCB design had issues with the GPIO_PIN_BACKPACK_BOOT pin oscillating.  I could not determine why in the hardware design, but removing this line helped.  It is also not required to set GPIO_PIN_BACKPACK_BOOT low and the pin is reset after the Backpack is power cycled.

Reply:
All good, seems pirate had the same problem on his DIY board